### PR TITLE
RUM release note 12-29 - getOverriddenServiceName

### PR DESCRIPTION
**Pull request by kimsauce**
## Purpose of this pull request

This pull request (PR) ...

Issue number: 

## Select the type of change:

- [ ] Minor Changes - Typos, formatting, slight revisions
- [x] Update Content - Revisions and updating sections
- [x] New Content - New features, sections, pages, tutorials
- [ ] Site and Tools - Updates, maintenance, and new packages for the site, Gatsby, React, etc

### CODE REVIEW

**Review by pbrzoska-sumo — APPROVED** [review]

**Review by njoy89 — APPROVED** [review]

**Review by MariePMcGarry — CHANGES_REQUESTED** [review]
@kimsauce Our standard has been to not use abbreviations like e.g, i.e., or .etc in docs.

**Comment by kimsauce** [comment]
@MariePMcGarry 
> Our standard has been to not use abbreviations like e.g, i.e., or .etc in docs.

fixed - can you pls re-review?

File: docs/apm/real-user-monitoring.md [hunk 1]
```diff
@@ -51,16 +51,16 @@ For full end-to-end visibility, we recommended supplementing your RUM browser au
 
 To configure a RUM HTTP Traces source:
 
-1. From Sumo Logic, select **Manage Data \> Collection \> Collection**. 
+1. From Sumo Logic, select **Manage Data** > **Collection** > **Collection**. 
 1. On the Collection page, click **Add Source** next to a Hosted Collector.
 1. Select **RUM HTTP Traces**. <br/><img src={useBaseUrl('img/rum/rum-icon.png')} alt="Real User Monitoring" width="120"/>
 1. Under **Source Type: RUM HTTP Traces**, enter the following information:
    * **Name** for the Source.
    * **Description** for the Source (optional).
```
Comment (stacykor) [inline docs/apm/real-user-monitoring.md:59]:
```suggestion
   * **Description**. (Optional) description of the Source .
```

File: docs/apm/real-user-monitoring.md [hunk 2]
```diff
@@ -51,16 +51,16 @@ For full end-to-end visibility, we recommended supplementing your RUM browser au
...
+   * **Application Name** (recommended): Add an **Application Name** tag of a text string to show for the app name in spans (for example, `bookings-app`). This groups services in the Application Service View. If left blank, services will belong to a "default" application. See [Application Service Dashboards](/docs/apm/traces/working-with-tracing-data/service-map.md) for more information. This setting is saved in the script for `name_of_your_web_application`.
+   * **Service Name** (required): Add a **Service Name** of a text string to show for the service name in spans (for example, `bookings-web-app`). This setting is saved in the script for `name_of_your_web_service`.<br/><br/>To set up a service name dynamically (e.g., to have different service names for micro-frontend packages), leverage the `getOverriddenServiceName` function inside your page code to overwrite the default service name (requires RUM script v4.2.0 or higher). Service names should be of low cardinality and should describe parts of your website above page level. Here's an example code leveraging that function:
```
Comment (MariePMcGarry) [inline docs/apm/real-user-monitoring.md:63]:
```suggestion
   * **Service Name** (required): Add a **Service Name** of a text string to show for the service name in spans (for example, `bookings-web-app`). This setting is saved in the script for `name_of_your_web_service`.<br/><br/>To set up a service name dynamically (for example, to have different service names for micro-frontend packages), leverage the `getOverriddenServiceName` function inside your page code to overwrite the default service name (requires RUM script v4.2.0 or higher). Service names should be of low cardinality and should describe parts of your website above page level. Here's an example code leveraging that function:
```

File: docs/apm/real-user-monitoring.md [hunk 3]
```diff
@@ -51,16 +51,16 @@ For full end-to-end visibility, we recommended supplementing your RUM browser au
...
 To configure a RUM HTTP Traces source:
 
-1. From Sumo Logic, select **Manage Data \> Collection \> Collection**. 
+1. From Sumo Logic, select **Manage Data** > **Collection** > **Collection**. 
 1. On the Collection page, click **Add Source** next to a Hosted Collector.
 1. Select **RUM HTTP Traces**. <br/><img src={useBaseUrl('img/rum/rum-icon.png')} alt="Real User Monitoring" width="120"/>
 1. Under **Source Type: RUM HTTP Traces**, enter the following information:
    * **Name** for the Source.
    * **Description** for the Source (optional).
    * **Source Host** and **Source Category** (optional): enter any string to tag the output collected from the source. These are [built-in metadata](/docs/search/get-started-with-search/search-basics/built-in-metadata.md) fields that allow you to organize your data. We recommend you specify a Source Category indicating the data is from a browser.<br/><img src={useBaseUrl('/img/rum/RUM-HTTP-Traces-Source.png')} alt="Real User Monitoring" width="300"/>
```
Comment (stacykor) [inline docs/apm/real-user-monitoring.md:60]:
```suggestion
   * **Source Host** and **Source Category** : (Optional) Enter any string to tag the output collected from the source. These are [built-in metadata](/docs/search/get-started-with-search/search-basics/built-in-metadata.md) fields that allow you to organize your data. We recommend you specify a Source Category indicating the data is from a browser.<br/><img src={useBaseUrl('/img/rum/RUM-HTTP-Traces-Source.png')} alt="Real User Monitoring" width="300"/>
```

File: docs/apm/real-user-monitoring.md [hunk 4]
```diff
@@ -51,16 +51,16 @@ For full end-to-end visibility, we recommended supplementing your RUM browser au
...
-   * **Service Name** (required): Add a **Service Name** of a text string to show for the service name in spans, for example `bookings-web-app`. This setting is saved in the script for `name_of_your_web_service`.<br/><br/>To set up a service name dynamically (e.g., to have different service names for micro-frontend packages), leverage the `getOverriddenServiceName` function inside your page code to overwrite the default service name (requires RUM script v4.2.0 or higher). Service names should be of low cardinality and should describe parts of your website above page level. Here's an example code leveraging that function:
+   * **Application Name** (recommended): Add an **Application Name** tag of a text string to show for the app name in spans (for example, `bookings-app`). This groups services in the Application Service View. If left blank, services will belong to a "default" application. See [Application Service Dashboards](/docs/apm/traces/working-with-tracing-data/service-map.md) for more information. This setting is saved in the script for `name_of_your_web_application`.
```
Comment (stacykor) [inline docs/apm/real-user-monitoring.md:62]:
```suggestion
   * **Application Name**. (Recommended) Add an **Application Name** tag of a text string to show for the app name in spans (for example, `bookings-app`). This groups services in the Application Service View. If left blank, services will belong to a "default" application. See [Application Service Dashboards](/docs/apm/traces/working-with-tracing-data/service-map.md) for more information. This setting is saved in the script for `name_of_your_web_application`.
```

File: docs/apm/real-user-monitoring.md (hunk 2 above)
Comment (stacykor) [inline docs/apm/real-user-monitoring.md:63]:
```suggestion
   * **Service Name**. ( Required) Add a **Service Name** of a text string to show for the service name in spans (for example, `bookings-web-app`). This setting is saved in the script for `name_of_your_web_service`.<br/><br/>To set up a service name dynamically (e.g., to have different service names for micro-frontend packages), leverage the `getOverriddenServiceName` function inside your page code to overwrite the default service name (requires RUM script v4.2.0 or higher). Service names should be of low cardinality and should describe parts of your website above page level. Here's an example code leveraging that function:
```

**Review by stacykor — CHANGES_REQUESTED** [review]
Not big, but not our usual style for Optional and Required. Just making it consistent.

**Pull request merged**